src/validation/User.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable require-jsdoc */
+import { HelperMethods } from '../utils';
+
+class Validation {
+  static Update(req, res, next) {
+    req
+      .checkBody('email')
+      .not()
+      .isEmpty()
+      .withMessage('Email is require to update role')
+      .trim()
+      .isEmail()
+      .withMessage('Invalid Email address');
+    req
+      .checkBody('role')
+      .not()
+      .isEmpty()
+      .withMessage('role is required')
+      .trim()
+      // eslint-disable-next-line max-len
+      .matches(/^(Super Administrator|Travel Administrator|Travel Team Member|Manager|Requester)$/)
```

Comment:
We don't need all this. just check for empty request body. I had told you to remove the express validator. please just check for empty req body. use what i did for validating the login route.